src/test/FreeC/Pass/ExportPassTests.hs

```diff
@@ -0,0 +1,94 @@
+-- | This module contains tests for "FreeC.Pass.ExportPass".
+
+module FreeC.Pass.ExportPassTests
+  ( testExportPass
+  )
+where
+
+import           Test.Hspec
+
+import           Data.Maybe                     ( fromJust )
+import qualified Data.Set                      as Set
+import qualified Data.Text                     as Text
+
+import qualified FreeC.Backend.Coq.Syntax      as Coq
+import           FreeC.Environment
+import           FreeC.Environment.Entry
+import           FreeC.Environment.ModuleInterface
+import           FreeC.Monad.Class.Testable
+import           FreeC.Monad.Converter
+import           FreeC.Pass.ExportPass
+import           FreeC.Pretty
+import           FreeC.Test.Environment
+import           FreeC.Test.Parser
+
+
+shouldBeQualifiedWith :: Coq.Qualid -> String -> Converter Expectation
+shouldBeQualifiedWith qualid modName = do
+  case qualid of
+    Coq.Qualified s _ -> return $ Text.unpack s `shouldBe` modName
+    Coq.Bare _ ->
+      return
+        $    expectationFailure
+        $    showPretty
+        $    prettyString "Expected qualifier"
+        <$$> line
+        <>   indent 2 (prettyString modName)
+        <>   line
+        <$$> prettyString "but it was not qualified"
+
+testExportPass :: SpecWith ()
+testExportPass = describe "FreeC.Pass.ExportPass" $ do
+  context "Exported entries are qualified when exported " $ do
+    it "Names of data types should be qualified when exported" $ do
```

Comment:
The description that is given to `it` should read like a sentence that describes what *it* (i.e., the `ExportPass`) is expected to do.

```suggestion
    it "qualifies Coq identifiers of data type entries" $ do
```